cmd/antrea-agent/config.go

```diff
@@ -120,4 +120,7 @@ type AgentConfig struct {
 	// Flow export frequency should be greater than or equal to 1.
 	// Defaults to "12".
 	FlowExportFrequency uint `yaml:"flowExportFrequency,omitempty"`
+	// Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig.
+	// Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
+	MasterOverride string `yaml:"masterOverride,omitempty"`
```

Comment:
Could you also update the config file here: https://github.com/vmware-tanzu/antrea/blob/ba78f3a2facf7edfa137d50064e5ef8a7416234c/build/yamls/base/conf/antrea-agent.conf
And `make manifest` to update related yamls.